projects/core/package.json

```diff
@@ -3,13 +3,15 @@
     "version": "3.3.0",
     "description": "The main zero-dependency and framework-agnostic Maskito's package to create an input mask",
     "keywords": [
+        "input",
```

Comment:
Add this keyword for other libraries too

https://github.com/taiga-family/maskito/blob/5640ed3a332b2816d8aad9fe083fb930769d7bef/projects/angular/package.json#L2-L5

https://github.com/taiga-family/maskito/blob/5640ed3a332b2816d8aad9fe083fb930769d7bef/projects/vue/package.json#L2-L5

https://github.com/taiga-family/maskito/blob/5640ed3a332b2816d8aad9fe083fb930769d7bef/projects/kit/package.json#L2-L5

https://github.com/taiga-family/maskito/blob/5640ed3a332b2816d8aad9fe083fb930769d7bef/projects/phone/package.json#L2-L5